tck/utils.py

```diff
@@ -0,0 +1,59 @@
+from hiero_sdk_python import Network
+from hiero_sdk_python.account.account_id import AccountId
+from hiero_sdk_python.client.client import Client
+from jsonrpcserver import Success, method, InvalidParams
+from hiero_sdk_python.crypto.private_key import PrivateKey
+
+__operatorAccountId: AccountId
+__operatorPrivateKey: PrivateKey
+__client: Client = Client(Network(network='testnet'))
+
+@method
+def setup(operatorAccountId: str=None, operatorPrivateKey: str=None, nodeIp: str=None, nodeAccountId: str=None, mirrorNetworkIp: str=None):
+    global __client, __operatorAccountId, __operatorPrivateKey
+    __operatorAccountId = AccountId.from_string(operatorAccountId)
+    __operatorPrivateKey = PrivateKey.from_string(operatorPrivateKey)
+    __client.set_operator(__operatorAccountId, __operatorPrivateKey)
+
+    return Success({
+        "message":"Successfully setup custom client.",
```

Comment:
```suggestion
        "message":"Successfully setup testnet client.",
```
This looks like it's only currently capable of setting up a testnet client. Custom client would use the `nodeIp`, `nodeAccountId`, and `mirrorNetworkIp` fields